### PR TITLE
[13.0] [MIG] purchase_exception

### PR DESCRIPTION
**Pull request by teewux**

### CODE REVIEW

File: purchase_exception/README.rst [hunk 1]
```diff
@@ -67,6 +67,7 @@ Contributors
 
 * Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
 * Sudhir Arya <sudhir@erpharbor.com>
+* Darius Martinkus <camptocamp.com>
```
Comment (HviorForgeFlow) [inline purchase_exception/README.rst]: missing <user>@

File: purchase_exception/models/__init__.py [hunk 2]
```diff
@@ -1,3 +1 @@
-from . import exception_rule
-from . import purchase
-from . import purchase_line
+from . import exception_rule, purchase, purchase_line
```
Comment (HviorForgeFlow) [inline purchase_exception/models/__init__.py]: It should be keeping in multiple lines, it does not hurt.

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
Code review

File: purchase_exception/wizard/purchase_exception_confirm.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+# Copyright 2020 Camptocamp SA
+# Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PurchaseExceptionConfirm(models.TransientModel):
+    _name = "purchase.exception.confirm"
+    _inherit = ["exception.rule.confirm"]
```
Comment (kittiu) [inline purchase_exception/wizard/purchase_exception_confirm.py:12]: `_description` is required to pass runbot.

**Comment by kittiu** [comment]
Functional Tested, LGTM

**Review by kittiu — APPROVED** [review]

File: purchase_exception/wizard/purchase_exception_confirm.py (hunk 3 above)
Comment (teewux) [reply purchase_exception/wizard/purchase_exception_confirm.py:12]: just added

**Review by Saran440 — APPROVED** [review]
functional test :+1: 
Please, Cloud you squash commit into 1?

**Comment by etobella** [comment]
@teewux Can you squas your commits in a single one, so we can proceed with this please :wink:

File: purchase_exception/models/purchase.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
+# Copyright 2017 Akretion (http://www.akretion.com)
+# Copyright 2020 Camptocamp SA
+# Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from odoo import api, models
+
+logger = logging.getLogger(__name__)
```
Comment (etobella) [inline purchase_exception/models/purchase.py:10]: This is not used

File: purchase_exception/models/purchase_line.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# Copyright 2017 Akretion (http://www.akretion.com)
+# Copyright 2020 Camptocamp SA
+# Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from odoo import api, fields, models
+
+logger = logging.getLogger(__name__)
```
Comment (etobella) [inline purchase_exception/models/purchase_line.py:10]: Not used

**Review by etobella — APPROVED** [review]
Code Review LGTM :+1:

**Comment by etobella** [comment]
@teewux If you don't have time, we can supersede the PR maintaining your ownership. WDYT?

**Comment by yvaucher** [comment]
This was superseeded by https://github.com/OCA/purchase-workflow/pull/1121

**Pull request closed without merging**